cmd/gobay/templates/Makefile.tmpl

```diff
@@ -102,7 +102,7 @@ build_debug: $(DIR_BUILD)
 	$(GO) build $(DEBUG_FLAGS) -o $(FILE_DEBUG_BIN) $(BIN_SRC)
 
 run: build_debug
-	./$(FILE_DEBUG_BIN) $(COMMAND) $(ARGS)
+	./$(FILE_DEBUG_BIN) $(COMMAND) $(ARGS) --env $(GOBAY_ENV) --root $(RUN)
```

Comment:
add default env and root, 
`make run COMMAND=grpcsvc ARGS="--root . --env development"` => `make run COMMAND=grpcsvc`